axiom-eth/src/rlp/tests.rs

```diff
@@ -394,6 +439,46 @@ mod rlp {
         }
     }
 
+    #[test] // fail
+    pub fn test_mock_rlp_array_1() {
+        let k = DEGREE;
+        let input_bytes: Vec<u8> = vec![0xc8, 0x83, b'c', b'a', b't', 0x83, b'd', b'o', b'g'];
+        let circuit = rlp_list_circuit(
+            RlcThreadBuilder::<Fr>::mock(), 
+            input_bytes, 
+            &[100, 100], 
+            true,
+        );
+        MockProver::run(k, &circuit, vec![]).unwrap().assert_satisfied();
+    }
+
+    #[test] // fail
```

Comment:
why is this a fail?

Reply:
The previous version failed this test -- deleting the comment.